docs/080-breaking-changes.rst

```diff
@@ -155,3 +156,15 @@ Interface Changes
   Use the "compact AST" (``--ast-compact--json`` resp. ``AST``) as replacement.
 
 * The old error reporter (``--old-reporter``) has been removed.
+
+
+How to update your code
+=======================
```

Comment:
Do you think it would be a good idea to say something about panics here?

One thing I noticed when fixing external tests was that unit tests that depend on `revert` returning a specific message on checked math errors now fail. That's not really a Solidity problem and more related to external support for the new codes but that's still something we might want to let people know about.

Reply:
Tooling often does not even display these messages because the eth rpc interface does not support it. It is a breaking change and listed above (I hope) but it's not really anything you can change your (Solidity) code to fix it...